parkings/models/permit.py

```diff
@@ -112,6 +114,8 @@ class Permit(TimestampedModelMixin, models.Model):
         'end_time': TimestampField(),
         'area': TextField(max_length=10),
     })])
+    # Meta
+    meta = JSONField(blank=True, null=True, encoder=DjangoJSONEncoder, verbose_name=_("meta"))
```

Comment:
The comment doesn't add any value. Better just remove it.

Reply:
```suggestion
    meta = JSONField(blank=True, null=True, encoder=DjangoJSONEncoder, verbose_name=_("meta"))
```

Reply:
Done